app/controllers/job_invocations_controller.rb

```diff
@@ -10,14 +10,34 @@ def new
         :bookmark_id => params[:bookmark_id]
       }
     }
-
-    if (template = JobTemplate.find_by(id: params[:template_id]))
+    template = (
+      JobTemplate.find_by(id: params[:template_id]) ||
+      RemoteExecutionFeature.find_by(label: params[:feature]).job_template
```

Comment:
maybe I'm missing something, but should it be handled by https://github.com/dLobatog/foreman_remote_execution/blob/7ea7ad9a909aca2461a18c030292f00d506a94f7/app/controllers/job_invocations_controller.rb#L110 ?

Reply:
:man_facepalming: